.github/CHANGELOG.md

```diff
@@ -773,16 +726,18 @@
 - Upgraded the documentation to use Sphinx 3.5.3 and the new m2r2 package.
   [(#1186)](https://github.com/PennyLaneAI/pennylane/pull/1186)
 
-- Added `flaky` as dependency for running tests in documentation. [(#1113)](https://github.com/PennyLaneAI/pennylane/pull/1113)
+- Added `flaky` as dependency for running tests in the documentation.
+  [(#1113)](https://github.com/PennyLaneAI/pennylane/pull/1113)
 
 <h3>Contributors</h3>
 
 This release contains contributions from (in alphabetical order):
 
-Shahnawaz Ahmed, Juan Miguel Arrazola, Thomas Bromley, Olivia Di Matteo, Kyle Godbey, Diego Guala, Josh Izaac,
+Shahnawaz Ahmed, Juan Miguel Arrazola, Thomas Bromley, Olivia Di Matteo, Alain Delgado Gran, Kyle
+Godbey, Diego Guala, Theodor Isacsson, Josh Izaac, Soran Jahangiri, Nathan Killoran, Christina Lee,
 Daniel Polatajko, Chase Roberts, Sankalp Sanand, Pritish Sehzpaul, Maria Schuld, Antal Száva.
```

Comment:
I went through the PR list so got all PR authors, but let me know if I am missing any code reviewers/other contributors!